cpp/modules/luma.gl/core/src/model.cpp

```diff
@@ -74,14 +75,14 @@ void Model::_initializeVertexState(ComboVertexStateDescriptor& cVertexState,
   cVertexState.vertexBufferCount = attributeSchema->num_fields();
 
   for (int location = 0; location < attributeSchema->num_fields(); location++) {
-    auto descriptor = attributeSchema->field(location)->descriptor();
+    auto format = attributeSchema->field(location)->type();
```

Comment:
Nit: I would consider the type being returned here. I think we may want this to be an "arrow type" and do the mapping here.

Litmus test:
- Is there any other place outside of this function where we need to know the mapping to wgpu constants?

Reply:
Not sure I follow -- you're suggesting for `garrow::Field::type` to return `arrow::DataType`? I'm not sure how we'd benefit from that, and mixing the two APIs would be confusing and have a large overhead due to complexity of `arrow::DataType`.

`garrow` currently has no dependency on `arrow`, other than stuff that's in `arrow-utils.h`, which is a set of functions that allow for easy mapping from one to the other, and could potentially be moved out.